DigitalLearningSolutions.Data/Services/NotificationService.cs

```diff
@@ -83,12 +37,12 @@ public void SendUnlockRequest(int progressId)
             }
 
             unlockData.ContactForename = unlockData.ContactForename == "" ? "Colleague" : unlockData.ContactForename;
-
-
-            var trackingSystemBaseUrl = configService.GetConfigValue(ConfigService.TrackingSystemBaseUrl);
+            var trackingSystemBaseUrl = configService.GetConfigValue(ConfigService.TrackingSystemBaseUrl) ??
```

Comment:
Is this error handled anywhere/what happens if this is thrown?

Reply:
No changes have been made as to how the NotificationService handles exceptions. Both times that exception is thrown in the NotificationService (and similarly when an exception is thrown in the EmailService that isn't explicitly handled within the function/controller) the generic "Something went wrong" page is shown. I think that is suitable for these - if not, do let me know.

Reply:
Yep, that sounds reasonable, thanks!